src/frontend/src/pages/HomePage/components/Task.tsx

```diff
@@ -3,19 +3,47 @@ import { useState } from 'react';
 import { KeyboardArrowRight, KeyboardArrowDown } from '@mui/icons-material';
 import SubtaskSection from './SubtaskSection';
 import { Checklist } from 'shared';
+import { useCheckedChecklists, useToggleChecklist } from '../../../hooks/onboarding.hook';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import ErrorPage from '../../ErrorPage';
+import { useToast } from '../../../hooks/toasts.hooks';
 
 interface SubtaskProps {
```

Comment:
The naming here is a bit confusing - I would think these would be the props for the inner component